weave/index/index_sqlite.py

```diff
@@ -293,13 +293,17 @@ def get_parents(self, basket_address, **kwargs):
         basket_address: str
             Argument can take one of two forms: either a path to the basket
             directory, or the UUID of the basket.
-        **kwargs unused for this function.
+        **max_gen_level: int (optional)
+            This indicates the maximum generation level that will be reported.
 
         Returns
         ----------
         pandas.DataFrame containing all the manifest data AND generation level
         of parents (and recursively their parents) of the given basket.
         """
+
+        max_gen_level = kwargs.get("max_gen_level", 100)
```

Comment:
If we default to a specific depth, we should keep them consistent (default in pandas is 999. Switch one of them so they all match)
Apply to get_children as well.

Reply:
Maybe this could be standardized in the parent class function.